docs/installation.rst

```diff
@@ -12,12 +12,8 @@ Run locally
 For development
 ---------------
 
-By default, *Kinto* persists the records and internal cache in a PostgreSQL
-database.
-
-The default configuration will connect to the ``postgres`` database on
-``localhost:5432``, with user/password ``postgres``/``postgres``.
-See more details below about installation and setup of PostgreSQL.
+By default, for convenience, *Kinto* persists the records, permissions and
+internal cache in a **volatile** memory backend.
```

Comment:
which means if you restart the server you loose your data and that if you start multiple process, you will have random behaviors.